runtime/src/main/java/org/corfudb/util/serializer/ProtobufSerializer.java

```diff
@@ -86,6 +86,13 @@ public Object deserialize(ByteBuf b, CorfuRuntime rt) {
             bbis.readFully(data);
             Record record = Record.parseFrom(data);
             Any payload = record.getPayload();
+            if (!classMap.containsKey(payload.getTypeUrl())) {
+                log.error("Deserialization error: Unknown type "+payload.getTypeUrl()+ " dumping map..\n");
+                for (String entry: classMap.keySet()) {
+                    log.error(entry + "=>" + classMap.get(entry));
```

Comment:
Same here?

Reply:
No, we want this line to be printed out even going forward, since it helps diagnose a tricky deserialization issue, so this is intentional!

Reply:
Okay, byw can you make it more descriptive?

Reply:
ok I will try